### PR TITLE
Exchage rmmh for katharine in top-level approvers

### PR DESCRIPTION
**Pull request by fejta**
/assign @stevekuznetsov @spiffxp

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -4,10 +4,10 @@ approvers:
 - cjwagner
 - fejta
 - ixdy
+- Katharine
```
Comment (BenTheElder) [inline OWNERS]: context: is currently oncall :-) :+1:

Comment (stevekuznetsov) [reply OWNERS]: But we will or will not remove after... ?

**Comment by stevekuznetsov** [comment]
Enjoy the auto-assignment woes, @Katharine 

/lgtm
/hold

File: OWNERS (hunk 1 above)
Comment (fejta) [reply OWNERS]: Planning to leave it until she removes herself

**Comment by stevekuznetsov** [comment]
lol
/woof

**Pull request merged**